packages/dashboard/src/components/tests/task-summary-panel.test.tsx

```diff
@@ -116,3 +116,38 @@ test('Get name of the actor from status', () => {
   const actor = getActorFromStatus(rawStatus);
   expect(actor).toEqual(['[tinyRobot/tinyRobot1]']);
 });
+
+describe('Components gets the correct label on specifics states', () => {
+  let task: RomiCore.TaskSummary;
+  beforeEach(() => {
+    task = Object.values(fakeTaskSummary())[0];
+  });
+
+  test('Shows ACTIVE label', () => {
+    task.state = RomiCore.TaskSummary.STATE_ACTIVE;
+    const root = mount(<TaskSummaryPanelInfo task={task} />);
+    expect(root.html()).toContain('ACTIVE');
+    root.unmount();
+  });
```

Comment:
Should we start doing this with react-testing-library? 

I think yes, we can install the library on this PR as well and start to refactor that too.

Reply:
https://github.com/osrf/romi-dashboard/issues/163